src/main/java/Duke.java

```diff
@@ -2,20 +2,33 @@
 import java.util.ArrayList;
 
 
+
 public class Duke {
     public static void main(String[] args) {
-        Scanner scannerObj = new Scanner(System.in);
+        showStartMessage();
```

Comment:
I like how you break your main() method into different functions corresponding to different stages of the code execution. Makes the code more readable and neat.